<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <property name="leftMargin">
     <number>3</number>
    </property>
    <property name="topMargin">
     <number>3</number>
    </property>
    <property name="rightMargin">
     <number>3</number>
    </property>
    <property name="bottomMargin">
     <number>3</number>
    </property>
    <property name="spacing">
     <number>2</number>
    </property>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QDockWidget" name="dockWidget">
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>2</number>
     </property>
     <property name="leftMargin">
      <number>3</number>
     </property>
     <property name="topMargin">
      <number>3</number>
     </property>
     <property name="rightMargin">
      <number>3</number>
     </property>
     <property name="bottomMargin">
      <number>3</number>
     </property>
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Parameters</string>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <property name="leftMargin">
         <number>3</number>
        </property>
        <property name="topMargin">
         <number>3</number>
        </property>
        <property name="rightMargin">
         <number>3</number>
        </property>
        <property name="bottomMargin">
         <number>3</number>
        </property>
        <property name="spacing">
         <number>2</number>
        </property>
        <item row="0" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QComboBox" name="comboBoxCMaps">
            <property name="sizePolicy">
             <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="currentIndex">
             <number>8</number>
            </property>
            <item>
             <property name="text">
              <string>Grayscale</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Hot</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Cold</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Night</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Candy</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Geography</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Ion</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Thermal</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Polar</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Spectrum</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Jet</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Hues</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="checkBoxInverted">
            <property name="text">
             <string>Inverted</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="checkBoxLog">
            <property name="text">
             <string>Log</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="checkBoxPeriod">
            <property name="text">
             <string>Period</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="1" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>N:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="spinBoxN">
            <property name="sizePolicy">
             <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>16</number>
            </property>
            <property name="singleStep">
             <number>1</number>
            </property>
            <property name="value">
             <number>3</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Max. iters:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="spinBoxMaxIters">
            <property name="sizePolicy">
             <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>999999999</number>
            </property>
            <property name="value">
             <number>1000</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="2" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Cx:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEditCx">
            <property name="sizePolicy">
             <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>0.285</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>Cy:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEditCy">
            <property name="sizePolicy">
             <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>0.01</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="3" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>Qx:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEditQx">
            <property name="sizePolicy">
             <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>0.0</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_9">
            <property name="text">
             <string>Qy:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEditQy">
            <property name="text">
             <string>-0.1</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="4" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Radius:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="doubleSpinBoxMaxNorm">
            <property name="maximum">
             <double>99999.000000000000000</double>
            </property>
            <property name="value">
             <double>2.000000000000000</double>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="comboBoxRanderModel">
            <property name="currentIndex">
             <number>0</number>
            </property>
            <item>
             <property name="text">
              <string>Normal Render</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Fast Render (x2)</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Very Fast Render (x3) </string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="checkBoxMandelbrot">
            <property name="text">
             <string>Mandelbrot</string>
            </property>
            <property name="checked">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="comboBoxFamily">
            <property name="currentIndex">
             <number>4</number>
            </property>
            <item>
             <property name="text">
              <string>Family00</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Family01</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Family02</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Family03</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Family04</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
        <item row="5" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <item>
           <widget class="QComboBox" name="comboBoxCouloring">
            <property name="currentIndex">
             <number>2</number>
            </property>
            <item>
             <property name="text">
              <string>escape vel.</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>escape vel. smoth</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string> orbit trap</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButtonRescale">
            <property name="text">
             <string>Rescale</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="checkBoxAutoRes">
            <property name="text">
             <string>Auto rescaling</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButtonAdjust">
            <property name="text">
             <string>Box home [-2,-2,2,2]</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="6" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_8">
          <item>
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>Orbitx:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEditOrbitX">
            <property name="text">
             <string>0.0</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Orbity:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEditOrbitY">
            <property name="text">
             <string>0.0</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="comboBoxOrbitType">
            <property name="currentIndex">
             <number>8</number>
            </property>
            <item>
             <property name="text">
              <string>x^2+y^2</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>|x|</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>|y|</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>|x*y|</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>x^2+y^2-2*x*y</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>x^2+y^2+2*x*y</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>x^2+y^2+|2*x*y|</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>x^2+y^2-|2*x*y|</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Min(|x|,|y|)</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Max(|x|,|y|)</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Line Dist.</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>|x-y|</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>|x+y|</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>|x|+|y|</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>|x|-|y|</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
        <item row="7" column="0">
         <widget class="DoubleSlider" name="horizontalSliderLo">
          <property name="minimum">
           <number>0</number>
          </property>
          <property name="maximum">
           <number>999999999</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="8" column="0">
         <widget class="DoubleSlider" name="horizontalSliderHi">
          <property name="minimum">
           <number>0</number>
          </property>
          <property name="maximum">
           <number>999999999</number>
          </property>
          <property name="value">
           <number>1000</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QPushButton" name="pushButtonSaveParams">
         <property name="text">
          <string>Save params</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton">
         <property name="text">
          <string>Load params</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonPNG">
         <property name="text">
          <string>Render to PNG</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBoxRegionSelection">
       <property name="text">
        <string>Region selection</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelFunction">
       <property name="palette">
        <palette>
         <active>
          <colorrole role="Text">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>0</red>
             <green>0</green>
             <blue>255</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="PlaceholderText">
           <brush brushstyle="SolidPattern">
            <color alpha="128">
             <red>0</red>
             <green>0</green>
             <blue>255</blue>
            </color>
           </brush>
          </colorrole>
         </active>
         <inactive>
          <colorrole role="Text">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>0</red>
             <green>0</green>
             <blue>255</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="PlaceholderText">
           <brush brushstyle="SolidPattern">
            <color alpha="128">
             <red>0</red>
             <green>0</green>
             <blue>255</blue>
            </color>
           </brush>
          </colorrole>
         </inactive>
         <disabled>
          <colorrole role="Text">
           <brush brushstyle="SolidPattern">
            <color alpha="127">
             <red>255</red>
             <green>255</green>
             <blue>255</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="PlaceholderText">
           <brush brushstyle="SolidPattern">
            <color alpha="128">
             <red>231</red>
             <green>231</green>
             <blue>231</blue>
            </color>
           </brush>
          </colorrole>
         </disabled>
        </palette>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DoubleSlider</class>
   <extends>QSlider</extends>
   <header>doubleslider.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
